### PR TITLE
i18n: look into core snaps when checking for translations

### PR DESCRIPTION
**Pull request by mvo5**

### CODE REVIEW

File: i18n/i18n.go [hunk 1]
```diff
@@ -49,24 +49,32 @@ func init() {
...
-		// ubuntu uses /usr/lib/locale-langpack and patches the glibc gettext
-		// implementation
-		langpack := filepath.Join(root, "..", "locale-langpack", r)
-		if osutil.FileExists(langpack) {
-			return langpack
+		// look into the core,ubuntu-core snaps first for
+		// translations, then the main system
+		candidateDirs := []string{
+			filepath.Join("/snap/core/current/", baseRoot),
+			filepath.Join("/snap/ubuntu-core/current/", baseRoot),
```
Comment (pedronis) [inline i18n/i18n.go]: should we drop this one now that we have the migration?

File: i18n/i18n_test.go [hunk 2]
```diff
@@ -127,3 +127,21 @@ func (s *i18nTestSuite) TestInvalidTextDomainDir(c *C) {
 	var Gtest = G
 	c.Assert(Gtest("singular"), Equals, "singular")
 }
+
+func (s *i18nTestSuite) TestLangpackResolver(c *C) {
```
Comment (pedronis) [inline i18n/i18n_test.go]: I'm a bit confused, is this testing the core cases?

Comment (niemeyer) [inline i18n/i18n_test.go]: Doesn't look like so..

File: i18n/i18n.go (hunk 1 above)
Comment (mvo5) [inline i18n/i18n.go]: Indeed, I dropped this now.

File: i18n/i18n_test.go (hunk 2 above)
Comment (mvo5) [inline i18n/i18n_test.go]: Sorry, this was indeed missing (only one of the two cases covered), I fixed this now.

**Pull request merged**